torchbenchmark/util/experiment/metrics.py

```diff
@@ -15,6 +15,8 @@
 @dataclasses.dataclass
 class TorchBenchModelMetrics:
     latencies: List[float]
+    cpu_peak_mem: float
```

Comment:
It should be Optional[float], not float.

Reply:
fixed by https://github.com/pytorch/benchmark/pull/1392/commits/57f0649d86e35507ceb9bc1608e1f588744faa90